src/status_im/contexts/wallet/create_account/view.cljs

```diff
@@ -133,7 +133,9 @@
                                                :emoji        @emoji
                                                :color        @account-color
                                                :path         @derivation-path
-                                               :account-name @account-name}]))
+                                               :account-name (if (string/blank? @account-name)
```

Comment:
Hey @ibrkhalil, I don't think we want this change. The real issue is that the slider button doesn't allow to disable and enable again. I have to create an issue for this and that is how we will handle it/ perhaps some design input. In future I think it's best to reach out and discuss things like this to save yourself the effort 
:)